edocumentation-service/src/main/java/org/dcsa/edocumentation/service/mapping/RequestedEquipmentMapper.java

```diff
@@ -9,7 +9,7 @@
 
 @Mapper(componentModel = "spring")
 public interface RequestedEquipmentMapper {
-  @Mapping(source = "booking", target = "booking")
+  @Mapping(source = "booking", target = "booking",ignore = true)
```

Comment:
If you are going to ignore booking then it shouldn't be part of the parameters any more. Also without changes to RequestedEquipmentService you have just broken both POST and PUT for bookings.

Reply:
Because booking is part of the task.